forest/daemon/src/daemon.rs

```diff
@@ -162,6 +161,23 @@ pub(super) async fn start(opts: CliOpts, config: Config) -> anyhow::Result<Db> {
     )?);
 
     chain_store.set_genesis(&genesis_header)?;
+    let db_garbage_collector = {
+        let db = db.clone();
+        let chain_store = chain_store.clone();
+        let get_tipset = move || chain_store.heaviest_tipset().as_ref().clone();
+        Arc::new(DbGarbageCollector::new(db, get_tipset))
+    };
+
+    #[allow(clippy::redundant_async_block)]
+    services.spawn({
+        let db_garbage_collector = db_garbage_collector.clone();
+        async move { db_garbage_collector.collect_loop_passive().await }
+    });
+    #[allow(clippy::redundant_async_block)]
+    services.spawn({
+        let db_garbage_collector = db_garbage_collector.clone();
+        async move { db_garbage_collector.collect_loop_event().await }
+    });
```

Comment:
It'd be nice to move it to a separate method. To my understanding, those calls in general need each other.

Reply:
`collect_loop_passive` and `collect_loop_event` work independently, either or both can be disabled